README.md

```diff
@@ -32,7 +32,7 @@ Please checkout [ML on GCP](https://github.com/GoogleCloudPlatform/ml-on-gcp) gu
 * [Inception on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception)
 * [Advanced Inception v3 on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception-v3-advanced)
 * [RetinaNet on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/retinanet)
-* [Transformer model on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/transformer) for natural-language translation, language modeling, and sentiment classification
+* [Transformer with Tensor2Tensor](https://cloud.google.com/tpu/docs/tutorials/transformer)
```

Comment:
Hmmm, now that it doesn't say "on Cloud TPU", but all the others do, it may make the reader think there's something different about this one. How about:

[Transformer with Tensor2Tensor on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/transformer)

Reply:
I now also realize I have "on TPU" duplicated in each entry. WDYT remove it? Or good to be super explicit in each?

Reply:
My 2c is that it's good to be super explicit in the case of Cloud TPU, since running on a TPU is exciting, and also perhaps a little uncommon for most people.